document/js-api/index.bs

```diff
@@ -736,6 +736,11 @@ enum WebAssemblyValueType {
   "f32",
   "f64"
 };
+</pre>
+
+Note: the WebAssemblyValueType enum is subject to future changes; several incomming WebAssembly GC proposals will add new values or extend the types. 
```

Comment:
I'd remove the word GC here; otherwise LGTM.

Reply:
Sorry Dan you are looking at an outdated version.

Currently we have:
> Note: this type may be extended with additional cases in future versions of WebAssembly. Client code should not assume that no other constants can occur.